examples/tutorials/analysis-time/time_resolved_spectroscopy.py

```diff
@@ -257,7 +257,6 @@ def time_resolved_spectroscopy(datasets, model, time_intervals):
 
 
 def create_table(time_intervals, fit_result):
-
```

Comment:
Should these notebooks be touched in this PR?

Reply:
This is because we renamed them with small caps so they went through ruff on commit.